### PR TITLE
Integrate design-tracker-template

### PR DESCRIPTION
**Pull request by michael-coker**
Integrated the design tracker template from https://github.com/rh-uxd/design-tracker-template

Verified it works with gh-pages on my fork - https://michael-coker.github.io/syndesis-ux/

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="description" content="Red Hat User Experience Design">
```
Comment (mindreeper2420) [inline _includes/header.html]: you'll want to change this description

File: _includes/header.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <meta name="author" content="Red Hat UXD">
```
Comment (mindreeper2420) [inline _includes/header.html]: change the author here as well

**Comment by michael-coker** [comment]
@mindreeper2420 thanks!

**Comment by amysueg** [comment]
Looks great! Thanks, @michael-coker and @mindreeper2420!
Now I guess we have to populate it... is it time for that training you were talking about @mindreeper2420 ?

**Pull request merged**